java/configuration/src/main/java/sleeper/configuration/table/index/DynamoDBTableIndex.java

```diff
@@ -112,6 +120,19 @@ public Stream<TableIdentity> streamAllTables() {
                 .sorted(Comparator.comparing(TableIdentity::getTableName));
     }
 
+    @Override
+    public Stream<TableIdentity> streamOnlineTables() {
+        return streamPagedItems(dynamoDB,
+                new ScanRequest()
+                        .withTableName(nameIndexDynamoTableName)
+                        .withConsistentRead(stronglyConsistent)
+                        .withFilterExpression("#Online = :online")
+                        .withExpressionAttributeNames(Map.of("#Online", ONLINE_FIELD))
+                        .withExpressionAttributeValues(Map.of(":online", createBooleanAttribute(true))))
+                .map(DynamoDBTableIdFormat::readItem)
+                .sorted(Comparator.comparing(TableIdentity::getTableName));
```

Comment:
This sorting will force loading all the tables into memory. This doesn't seem necessary?